depends/packages/native_capnp.mk

```diff
@@ -1,9 +1,13 @@
 package=native_capnp
-$(package)_version=0.7.0
+$(package)_version=1.0.1
 $(package)_download_path=https://capnproto.org/
 $(package)_download_file=capnproto-c++-$($(package)_version).tar.gz
 $(package)_file_name=capnproto-cxx-$($(package)_version).tar.gz
-$(package)_sha256_hash=c9a4c0bd88123064d483ab46ecee777f14d933359e23bff6fb4f4dbd28b4cd41
+$(package)_sha256_hash=0f7f4b8a76a2cdb284fddef20de8306450df6dd031a47a15ac95bc43c3358e09
+
+define $(package)_set_vars
+  $(package)_config_opts=--with-openssl=no
```

Comment:
Should this be added to the non-native package too?

Nit: `--without-openssl`?